### DISCUSSION TITLE
关于评估模型的间隔设定

### DISCUSSION THREAD

**Discussion by wq881103** [PaddlePaddle/PaddleOCR#13080] [Q&A]
下载的默认配置文件中设定的[0,400]，表示的意思应该是训练开始时先评估一次，可是开始那一次评估并没有发生，是在step 400以后才评估的，这是怎么回事？

**Comment by GreatV** [comment]
请参考这份文档 https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_ch/config.md#global
表示从0开始每隔400 step 评估一次
